src/test/java/org/jenkins/tools/test/hook/JacocoHookTest.java

```diff
@@ -54,28 +54,18 @@ public void testAction() {
         info.put(
                 "args",
                 new ArrayList<>(
-                        List.of(
-                                "--define=forkCount=1",
-                                "hpi:resolve-test-dependencies",
-                                "hpi:test-hpl",
-                                "surefire:test")));
+                        List.of("hpi:resolve-test-dependencies", "hpi:test-hpl", "surefire:test")));
```

Comment:
Adapting to the fact that we're no longer in the business of managing `forkCount`.